values.yaml

```diff
@@ -109,6 +111,19 @@ configData:
     addr: :5000
     headers:
       X-Content-Type-Options: [nosniff]
+    ## The debug interface configuration
+    ##
+    ## Will be autocreated if metrics.enabled is true
+    ## Set values here to override the defaults
+    # debug:
+      ## The address and port the debug server should listen on
+      # addr: 0.0.0.0:5001
+      ## Prometheus metrics specific configuration
+      # prometheus:
+          ## Do not disable this if you use metrics.enabled: true
+          # enabled: true
+          ## Make sure to adjust metrics.serviceMonitor.path accordingly
+          # path: /metrics
```

Comment:
Just thinking aloud and paraphrasing to make sure I understand how this works:

 * To enable metrics, all a user has to do is set `metrics.enabled: true` in the values provided to this chart. If that is set, then a default configuration (as described above) will be injected into the registry's configuration to activate the debug interface and expose it on a default port.
 * If `metrics.enabled: true` is set, and `configData` already has a debug interface configuration, then this chart will derive the relevant port/path/etc from that registry config.

If those assumptions are correct, that seems like reasonable behavior to me.